tilecloud_chain/tests/tilegeneration/test.yaml

```diff
@@ -63,10 +63,8 @@ caches:
         folder: tiles
 
 defaults:
-    layer: &layer
+    all_layer: &all_layer
```

Comment:
I would use different naming :
- `all_layer` => `layer` or `base_layer`
- `layer` => `mapserver_layer`

`all_layer` make no sense (`one_layer` or `all_layers`, but cannot be `all_layer`).